src/videojs-http-streaming.js

```diff
@@ -60,6 +60,7 @@ const Vhs = {
   'GOAL_BUFFER_LENGTH_RATE',
   'BUFFER_LOW_WATER_LINE',
   'MAX_BUFFER_LOW_WATER_LINE',
+  'MAX_BUFFER_LOW_WATER_LINE_NEW',
```

Comment:
It may be worth a comment somewhere explaining more what this is (or a different name instead of the _NEW suffix).

Reply:
It's in the other pr https://github.com/videojs/http-streaming/pull/886/files#diff-9d9a6cd82f41984872a66a3ab0d440c4R15

Reply:
yeah, I think eventually we should probably rename it to like `EXPERIMENTAL_LOW_WATER_LINE` or something like that but that can be done later.